.github/workflows/go.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow will build a golang project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
+
+name: Go
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
```suggestion
permissions: {}

jobs:

  build:
    runs-on: ubuntu-latest
    permissions:
      contents: read
    steps:
```

Reply:
Done

Reply:
you're still missing the "permissions" block here which overrides the default (which is to give none)

Reply:
I think the workflow still works without any permissions? [This run](https://github.com/google/sbom-conformance/actions/runs/14254258865/workflow) succeeded. Looking at the [docs](https://docs.github.com/en/rest/repos/contents?apiVersion=2022-11-28#get-repository-content), it says that that some endpoints can be used without permissions if only public resources are requested. But maybe there are other reasons to add this, I'm not sure.

Reply:
TIL , thanks!